### DISCUSSION TITLE
soooooooooooooooooooooooooooooo hows it goin?

### DISCUSSION THREAD

**Discussion by Andlesto** [phst-randomizer/ph-randomizer#438] [Q&A] [answered]
recently started playing the ds Zelda's and i absolutely love them looked online for a randomizer and there isnt one for either???? this one seems to be the most worked on sooooooo hows it goin so far?

**Comment by mike8699** [comment, marked as answer]
Things have slowed down recently, but it's still in development. The DS Zeldas are generally not as popular as the rest of the series, and that sentiment more or less translates to less technical information about the game being available as not many people have attempted to make ROM hacks for them. Meaning there's a lot of reverse engineering work involved on top of the usual technical work required for a randomizer, not all of which is strictly related to the randomizer so it's not visible on the github repo here.
